js/chat/chat.scss

```
@@ -95,6 +96,17 @@ shiny-chat-message {
       max-height: 66%;
       max-width: 66%;
     }
+
+    // Provide .border-0 as a way to opt-out of border container
+    &:has(> .border-0) {
+      border: none;
+      border-radius: unset;
+      overflow: unset;
+    }
+    > .border-0 {
+      max-height: 100%;
+      max-width: 100%;
+    }
```

I think, with `> img` removed from the above rule, this can be

```suggestion
    // Provide .border-0 as a way to opt-out of border container
    &:has(> .border-0) {
      border: none;
      border-radius: unset;
      overflow: unset;
    }
```

but this did point out to me that we should probably add `max-height` and `max-width` to the `> *` rule above so that inline height/width on image have a constraint.